5.test.js

```diff
@@ -0,0 +1,7 @@
+const assert = require('assert')
+
+const disemvowel = require('./5')
+
+assert(disemvowel('i am groot') === 'mgrt')
+
+console.log('All tests passed')
```

Comment:
Yes, lol.  If the expression inside the assert evaluates to `false` an error will be thrown and line 7 will never be run.